packages/plugin-credentials/src/credentials/common.js

```diff
@@ -201,14 +208,14 @@ export default {
     return Reflect.apply(SparkPlugin.prototype.initialize, this, ...args);
   },
 
-  @waitForValue(`@`)
+  @waitForValue(`isLoggingOut`)
```

Comment:
Logout should not be called until `@` is loaded and `isLoggingOut` should not be persisted (make it a session variable prevent it from being persisted).